.gitignore

```diff
@@ -25,6 +25,8 @@ CMakeFiles
 *.o
 *.obj
 compile_commands.json
+**/Output
+**/lit.site.cfg.py
```

Comment:
Are these files created during the build?   Where are they created exactly?

Reply:
This file is generated at build configuration time by CMake. For a normal in-tree build, this is created at ./test/lit/lit.site.cfg.py, but it's relative to the CMake build directory rather than the source directory.

Reply:
Got it.  In that case it should just be `/test/lit/lit.site.cfg.py` since it has a fixed location.